### DISCUSSION TITLE
Closh: Anyone done this on closh?

### DISCUSSION THREAD

**Discussion by SSYH** [starship/starship#1502] [Q&A]
I just discovered **closh** yesterday and **starship** today...
Curious whether anyone has this same combination...?
Thanks (TIA)

**Comment by vladimyr** [comment]
@SSYH You mean https://github.com/dundalek/closh?

There are already a few shell support requests, feel free to create one for closh. :wink:

**Comment by chipbuster** [comment]
According to [this](https://github.com/dundalek/closh/blob/master/doc/guide.md), you can probably get a basic starship prompt by defining

```
(defn closh-prompt []
  (sh-str starship prompt))
```

but this won't cover things like execution time or any mode indicators.
